src/librustc/mir/mod.rs

```diff
@@ -63,7 +63,7 @@ macro_rules! newtype_index {
 }
 
 /// Lowered representation of a single function.
-// Do not implement clone for Mir, its easy to do so accidently and its kind of expensive.
+/// Do not implement clone for Mir, which can be accidently done and kind of expensive.
```

Comment:
Looks good!

Regarding your `//` to `///` change, I'm not sure if this information belongs in the doc comment. What do you think?